less/mixins/text-emphasis.less

```diff
@@ -2,7 +2,7 @@
 
 .text-emphasis-variant(@color) {
   color: @color;
-  a&:hover {
+  a&:hover, a&:focus {
```

Comment:
If we did this, this new selector would need to appear on the next line like so:

```less
a&:hover,
a&:focus {
```